### PR TITLE
Send email to oncall when form is completed

### PR DESCRIPTION
**Pull request by whitdog47**
In this PR, an additional field for oncall service is added to the form type. When a form of this form type is submitted, if provided, the current oncall will receive an email notification about the completed form.

### CODE REVIEW

**Review by anukaal — APPROVED** [review]

File: src/dispatch/incident/messaging.py [hunk 1]
```diff
@@ -217,6 +219,55 @@ def send_welcome_email_to_participant(
...
 
+def send_completed_form_email(
+    participant_email: str, form: Forms, db_session: SessionLocal
+):
+    """Sends an email to notify about a completed incident form."""
+    plugin = plugin_service.get_active_instance(
+        db_session=db_session, project_id=form.project.id, plugin_type="email"
+    )
+    if not plugin:
+        log.warning("Completed form notification email not sent, not email plugin configured.")
```
Comment (metroid-samus) [inline src/dispatch/incident/messaging.py:230]:
```suggestion
        log.warning("Completed form notification email not sent. No email plugin configured.")
```

File: src/dispatch/messaging/strings.py [hunk 2]
```diff
@@ -175,6 +176,16 @@ class MessageType(DispatchEnum):
 Description of the actions taken to resolve the case.
 """.replace("\n", " ").strip()
 
+INCIDENT_COMPLETED_FORM_DESCRIPTION = """
+ A new {{form_type}} form related to incident {{name}} has been
+ submitted that requires your immediate attention. This form details
```
Comment (metroid-samus) [inline src/dispatch/messaging/strings.py:181]:
```suggestion
        submitted and requires your immediate attention. This form details
```

File: src/dispatch/static/dispatch/src/forms/types/NewEditSheet.vue [hunk 3]
```diff
@@ -67,6 +67,12 @@
                   name="Form Schema"
                 />
               </v-col>
+              <v-col cols="12">
+                <span class="text-body-1 text-medium-emphasis mt-2">
+                  Send an email to this oncall when a form is submitted as complete.
+                </span>
+                <service-select label="Oncall Service" :project="project" v-model="service" />
```
Comment (metroid-samus) [inline src/dispatch/static/dispatch/src/forms/types/NewEditSheet.vue:74]: This should be an optional field.

File: src/dispatch/forms/type/service.py [hunk 4]
```diff
@@ -35,10 +40,15 @@ def create(*, db_session: Session, forms_type_in: FormsTypeCreate, creator) -> F
         db_session=db_session, email=creator.email, project_id=project.id
     )
 
+    service = service_service.get(
```
Comment (metroid-samus) [inline src/dispatch/forms/type/service.py:43]: This field should be optional.

**Review by metroid-samus — APPROVED** [review]

**Review by metroid-samus — CHANGES_REQUESTED** [review]
Requesting changes

**Review by metroid-samus — APPROVED** [review]

**Pull request merged**